cumulusci/core/config/MergedConfig.py

```diff
@@ -0,0 +1,96 @@
+import io
+import yaml
+
+from builtins import bytes, int, str
+from collections import OrderedDict
+
+from cumulusci.core.config import BaseConfig
+from cumulusci.core.exceptions import ConfigMergeError, ConfigError
+
+
+class MergedConfig(BaseConfig):
+    """ A merged config takes a collection of dicts and merges them in priority order."""
+
+    def __init__(self, **configs):
+        """ MergedConfig(user_config={}, base_config={},) """
+        self.configs = (
+            configs
+        )  # NOTE: This relies on Py3.6+ behavior, kwarg order is preserved.
```

Comment:
figure out what to do about this.

Reply:
Either pass item tuples or a whole ordereddict without kwargifying it